packages/features/portfolio/src/containers/portfolio-state-zero/hooks/use-card-data.tsx

```diff
@@ -10,18 +10,27 @@ import {
   VaultIcon,
 } from '@notional-finance/icons';
 import { useSelectedNetwork } from '@notional-finance/notionable-hooks';
-import { PORTFOLIO_STATE_ZERO_OPTIONS } from '@notional-finance/util';
+import { PORTFOLIO_STATE_ZERO_OPTIONS, PRODUCTS } from '@notional-finance/util';
 import { FormattedMessage } from 'react-intl';
 
 export const useCardData = (
   selectedTab: number,
   activeToken: string,
-  activeTokenData: any[],
-  leveragedTokenData: Record<string, any>
+  activeTokenData: any[]
 ) => {
   const theme = useTheme();
   const selectedNetwork = useSelectedNetwork();
 
+  const leveragedLiquidityData = activeTokenData?.find(
+    (data) => data.product === 'Leveraged Liquidity'
+  );
+  const leveragedYieldFarming = activeTokenData?.find(
+    (data) => data.product === 'Leveraged Vault'
+  );
+  const leveragedPointsFarming = activeTokenData?.find(
+    (data) => data.product === 'Leveraged Vault' && data.pointMultiples
```

Comment:
I wonder if we should change `data.product` instead of having this pointMultiples logic here. Is that going to be a big refactor?

Having data.product be "Leveraged Yield Farming" or "Leveraged Point Farming"

Reply:
The product here is coming from the allYields object in global state. Updating that would break a lot of stuff.